ingesters/s3.md

```diff
@@ -0,0 +1,113 @@
+# Amazon Simple Storage Service Ingester (S3)
+
+The Amazon S3 ingester is designed to ingest data directly from the Amazon S3 API.  While this is a convenient method of ingestion and integrates well with other tools like Amazon Cloudtrail and a myriad of other cloud based tools, be aware that this ingester is NOT designed to provide near real-time access to data.  Nor can it "tail" objects, consuming entries as they are written to a bucket.  The Amazon S3 system is a blob storage system with three basic operations: `list`, `get`, `put`;  this means that the ingester must consume entire objects when ingesting.  If new data is written to an S3 object, it is essentially a replacement of the object and the ingester will treat it as such.
+
+S3 buckets are also not meant to provide low latency access to data. It is basically an HTTP API and most of the systems that write to S3 buckets batch up data and write large chunks at a time.  Be forewarned, you may see delays measured in minutes between when an entry says it was created and when Gravwell can actually consume it.  That being said, the S3 ingester makes integrating with many cloud services extremely easy: just point at the S3 bucket and provide the appropriate access keys and you are off.
```

Comment:
"It is basically an HTTP API"
"It" is pretty vague here. Consider revising

Reply:
updated to be "The S3 API is basically an HTTP API"